packages/perseus/src/util/graphie.ts

```diff
@@ -1445,6 +1440,16 @@ export class Graphie {
             "addToVisibleLayerWrapper is not ready. Call addMouseLayer() first.",
         );
     }
+
+    // This is a stub that's overridden in interactive.ts
+    getMouseCoord(event: Readonly<{pageX?: number; pageY?: number}>): Coord {
+        throw new Error("getMouseCoord is a stub, and is not implemented");
+    }
+
+    // This is a stub that's overridden in interactive.ts
+    getMousePx(event: Readonly<{pageX?: number; pageY?: number}>) {
+        throw new Error("getMousePx is a stub, and is not implemented");
+    }
```

Comment:
I just moved these down below the two functions that are related to `addMouseLayer` so they are all together.